### DISCUSSION TITLE
iCub: usb system key unplugged

### DISCUSSION THREAD

**Discussion by pfdominey** [robotology/community#266] [Q&A] [answered]
The USB connector for the USB disk for the iCub pc104 came unplugged, and I simply do not know where it plugs back.  in the attached photo, the USB key is hanging down from the red arrow (so we cant see it but it is there), the connector that came uplugged is marked by thewhite arrow, and I think it plugs in somewhere near the yellow arrow but not sure.

can you please help me to find where to re-plug this?  should be quite simple for someone who knows :)
[iCubUSBConnection.pdf](https://github.com/robotology/QA/files/1542271/iCubUSBConnection.pdf)

**Comment by pattacini** [comment, marked as answer]
Hi @pfdominey 

The picture below should give you the heads-up

![2af84f4a-1e59-4549-8cd8-e9868b7a9c0b](https://user-images.githubusercontent.com/3738070/33762301-45366fa2-dc0c-11e7-8312-6287fc02b617.jpg)

Thanks @vtikha for the photo.

**Comment by pfdominey** [comment]
Perfect!  Thanks very much

**Comment by pfdominey** [comment]
Beautiful photo! Thanks Ugo

On Fri, Dec 8, 2017 at 11:39 AM, Ugo Pattacini <notifications@github.com>
wrote:

> Hi @pfdominey <https://github.com/pfdominey>
>
> The picture below should give you the heads-up
>
> [image: 2af84f4a-1e59-4549-8cd8-e9868b7a9c0b]
> <https://user-images.githubusercontent.com/3738070/33762301-45366fa2-dc0c-11e7-8312-6287fc02b617.jpg>
>
> Thanks @vtikha <https://github.com/vtikha> for the photo.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/robotology/QA/issues/266#issuecomment-350231782>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AepGxqxNY-Kvn26P9nNGTbvXJzol8pLkks5s-RHpgaJpZM4Q64Xf>
> .
>



-- 
Peter Ford Dominey PhD, CNRS Research Director
Inserm U846
Stem Cell and Brain Research Institute
Integrative Neuroscience & Robotics
18, avenue du doyen Jean Lépine
69675 Bron cedex
France
Tél : +33-4 72 91 34 84
Fax : +33-4 72 91 34 61
http://www.sbri.fr/teams/human-and-robot-interactive-cognitive-systems.html
http://www.sbri.fr
